.docs/content/0.armonik/components/3.armonik/2.compute-plane/1.polling-agent.md

```diff
@@ -0,0 +1,5 @@
+# Polling agent
```

Comment:
Shouldn't we rename it ?

Reply:
We will be able to rename it when it will be done in core